pkg/api/client_test.go

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2021 The Sigstore Authors.
```

Comment:
```suggestion
// Copyright 2022 The Sigstore Authors.
```

Reply:
Ah right we should update the boilerplate everywhere and the boilerplate test to match...

Reply:
no, this is added when you create the file first time. 

if is a new file you add the current year